packages/venia-ui/lib/components/CategorySort/categorySort.js

```diff
@@ -88,13 +89,7 @@ CategorySort.propTypes = {
             sortDirection: sortDirections
         })
     ),
-    sortControl: shape({
-        currentSort: shape({
-            sortAttribute: string,
-            sortDirection: sortDirections
-        }),
-        setSort: func.isRequired
-    })
+    sortProps: array
```

Comment:
Same feedback here. Would be useful to go in more detail on this propType.

Reply:
I'm the one who added this line, since we changed the return signature of the hook. I know the following will work, but I don't know how to specifically indicate we expect an array of length `2` in which the first element is an object and the second is a function.

```es6
sortProps: arrayOf(
  oneOf(
    shape({}),
    func
  )
)
```

Reply:
Ok, I've asked @larsroettig to add a comment instead.